oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/SuperDataset.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.storage.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apache.skywalking.oap.server.core.analysis.manual.segment.SegmentRecord;
+
+/**
+ * This annotation applies to super size dataset entities
+ * for example:
+ * <p>1. adjust the number of shards in es storage</p>
+ *
+ * @see SegmentRecord
+ * @see org.apache.skywalking.oap.server.storage.plugin.zipkin.ZipkinSpanRecord
+ * @see org.apache.skywalking.oap.server.storage.plugin.jaeger.JaegerSpanRecord
```

Comment:
These should be removed. This is core level annotation, those entities are only optional.